message_definitions/v1.0/messages.xml

```diff
@@ -2208,8 +2208,14 @@
       <field name="yaw_c"  type="float"/>
   </message>
 
-  <!-- 251 is free -->
-  <!-- 252 is free -->
+   <!--251 is free -->
+
+   <message name="GPS_SMALL" id="252">
+    <field name="multiplex_speed"  type="uint32">bits 31-21 course in decideg : bits 20-10 ground speed in cm/s : bits 9-0 climb speed in cm/s</field>
+    <field name="lla_lat"  type="int32" unit="1e7deg" alt_unit="deg" alt_unit_coef="0.0000001"></field>
```

Comment:
as already mentioned in the other PR, I think it makes sense to keep the `lat` and `lon` field names like in all the other messages or is there a reason for naming them `lla_lat` and `lla_lon` here?

Same for ACINFO_LLA